src/common/src/csv_parser.cc

```diff
@@ -62,11 +63,19 @@ types::Error CsvParser::Parse(const std::string &input_string)
         records_.push_back(Split(line, CSVPARSER_VALUE_DELIMITER));
     }
 
+
+    // Restriction
     const std::size_t size = records_[0].size();
     for (const std::vector<std::string> &record : records_)
     {
         if (record.size() != size)
         {
+            std::cout << "Error at record: "; 
```

Comment:
Not sure we want to `cout` in the parser.  We can talk about some other ways to get the record(s) that fails the size check if that is something you need.  If you only need to know that parsing failed, we can remove the `cout` statements and `iostream` include.

Reply:
agreed cout statements and iostream include removed